### DISCUSSION TITLE
Planetary Gears - Too Loose

### DISCUSSION THREAD

**Discussion by hamtroniks** [almtzr/Pedro#6] [Q&A]
I decided to test print a few pieces while I wait to (hopefully) get the STEP files. I started with the planetary gear assembly & the center gear/axle is WAY too loose to make contact with the outer gears. You can manipulate it to engage with 1 or maybe 2, but that leaves the 3rd completely free. 

What printer & nozzle size did you use for these parts? I doubt that it's specific to my machine (Bambu Labs) as the play is far too much to tighten up with just settings alone.

**Comment by almtzr** [comment]
Hi, I just upgraded the planetary gear system and uploaded the following files below. 
Let me know if you have any issues with them.

- 8-arm_gear_v2.2.stl
- 13-planetary_gear_cover_v2.2.stl
- 14-planetary_carrier_v2.0.stl
- 15-sun_gear_v2.0.stl
- 16-planet_gear_v2.0.stl

**Reply by hamtroniks** [reply]
Awesome!! I'll check them out now.

**Comment by almtzr** [comment]
The STL files have been updated in accordance with the Kickstarter campaign.
